### DISCUSSION TITLE
[Answer sanitization check] All 0 and 1

### DISCUSSION THREAD

**Discussion by alinghi** [bcaitech1/p4-dkt-lastffang#1] [Submissions]
1) 모두 0
Accuracy 0.5269
AUROC 0.5000

2) 모두 1
Accuracy 0.4731
AUROC 0.5000

**Comment by GY-Jeong** [comment]
training data는 1의 분포가 67%정도였는데 실제 분포는 다르네요!

**Reply by newspring97** [reply]
training data와 분포가 다르다면 Private leaderboard 상에서의 분포도 다를 수 있을 것 같아요!
